feat: add Marketing Analyst role to jobs page (#469)
ory/web: src/markdown/jobs/marketinganalyst.mdx

---
published: true
path: '/jobs/marketing-analyst/'
title: 'Marketing Analyst'
metaTitle: 'Open Marketing Careers at Ory'
metaDescription: 'Read the Marketing Analyst job description.'

> lastUpdatedAt: 'December 12, 2021'

Reviewer: ```suggestion
lastUpdatedAt: 'December 17, 2021'
```

- **Location**: Munich, Germany
- **Components**: Google Analytics (data Studios), Hubspot, CMS platforms

### What You'll Do

- Conceptualize and perform digital marketing analysis about the following topics: Ory customer journey (onsite), target groups & prospective customers, marketing efficiency (media efficiency, attribution modeling), competition
- Perform marketing reporting and data preparation. Define KPIs (ROI, ARR, MRR) and create standardized/automated reports (Google Data Studios) and dashboards linking different data sources to derive marketing strategies
- Monitor and evaluate campaign performance, A/B testing experiments, developing insights on how to improve impact. This includes determining KPIs, testing hypotheses, forecasting campaigns, and communicating insights to relevant members of the organization. 
- Keep track of data that will inform marketing strategy and budgeting. 
- Communicate across departments (i.e. sales, engineering, HR) to keep them informed about relevant marketing performance data and analytics.

### We’re looking for someone who has

- A degree in economics, mathematics, business informatics, consumer insights or a similar field.

> - 5 + years of professional experience in data science, analytics, and marketing BI 

Reviewer: ```suggestion
- 4 + years of professional experience in data science, analytics, and marketing BI 
```

---
published: true
path: '/jobs/marketing-analyst/'
title: 'Marketing Analyst'
metaTitle: 'Open Marketing Careers at Ory'
metaDescription: 'Read the Marketing Analyst job description.'
lastUpdatedAt: 'December 12, 2021'
description:
  'Conceive and perform marketing analysis to grow the Ory brand.'
location: 'Munich, Germany'
position: 'Full Time'
---

import { About, Dev, WorkAtOry } from './shared'

## Marketing Analyst

<About />

Ory is the largest open-source authentication and authorization (login,
registration, MFA, permissions, OAuth2) ecosystem and community in the world. We
develop open-source software and use it to build a scalable, easy-to-use, and
secure cloud platform. You will provide organizational support to Ory leadership
to ensure smooth operations and communications. Together with the CEO, you will
provide a core leadership function that collaborates with UX design,
engineering, finance, marketing, and data analytics.

### Overview

- **Position Type:** Full Time
- **Location**: Munich, Germany

> - **Components**: Google analytics, Hubspot, CMS platforms

Reviewer: ```suggestion
- **Components**: Google Analytics (data Studios), Hubspot, CMS platforms
```

### What You'll Do

> - Conceive and perform marketing analysis about the following topics: Ory customer journey, target groups & prospective customers, marketing efficiency (media efficiency, attribution modeling), competition

Reviewer: ```suggestion
- Conceptualize and perform digital marketing analysis about the following topics: Ory customer journey (onsite), target groups & prospective customers, marketing efficiency (media efficiency, attribution modeling), competition
```

> - Perform marketing reporting and data preparation. Define KPIs and create standardized / automated reports and dashboards for marketing and linking of different data sources to derive marketing strategies

Reviewer: ```suggestion
- Perform marketing reporting and data preparation. Define KPIs (ROI, ARR, MRR) and create standardized/automated reports (Google Data Studios) and dashboards linking different data sources to derive marketing strategies
```

- Monitor and evaluate campaign performance, A/B testing experiments, developing insights on how to improve impact. This includes determining KPIs, testing hypotheses, forecasting campaigns, and communicating insights to relevant members of the organization. 
- Keep track of data that will inform marketing strategy and budgeting. 
- Communicate across departments (i.e. sales, engineering, HR) to keep them informed about relevant marketing performance data and analytics.

### We’re looking for someone who has

- A degree in economics, mathematics, business informatics, consumer insights or a similar field.
- 5 + years of professional experience in data science, analytics, and marketing BI 

> - An understanding of data-driven marketing, customer journeys, traditional and digital marketing channels

Reviewer: ```suggestion
- An understanding of data-driven marketing, customer journeys and digital marketing channels
```

> - Experience with marketing, analytics, reporting tools, and CMS platforms (i.e. Google analytics) 

Reviewer: ```suggestion
- Experience with marketing, analytics, reporting tools, and CMS platforms (i.e. Google Analytics, data Studios) 
```

> - Facility with G Suite tools and knowledge of SQL.

Reviewer: ```suggestion
```
